src/apps/loan-list/list/loan-list.tsx

```diff
@@ -223,7 +224,7 @@ const LoanList: FC = () => {
             <Pagination
               dueDates={dueDates}
               loans={loans}
-              view={view}
+              view={view as ListView}
```

Comment:
It should be possible to set the ListView type [here](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/2637492e9d9db2729ed5a651823c0296c657f3c8/src/apps/loan-list/list/loan-list.tsx#L60) instead. And then you wouldn't need the type assertion.

Reply:
Done